src/extension/popup-page/index.tsx

```diff
@@ -99,6 +100,8 @@ function PopupUI() {
     const [, setOpen] = useRemoteControlledDialog<MaskbookWalletMessages, 'selectProviderDialogUpdated'>(
         WalletMessageCenter,
         'selectProviderDialogUpdated',
+        noop,
```

Comment:
you don't need to add a module-level variable reference to the dependencies list.